### DISCUSSION TITLE
I think I found a bug

### DISCUSSION THREAD

**Discussion by almodovaris** [official-stockfish/Stockfish#4537] [Q&A]
1. e4 e5 {(e7-e5 Ng1-f3 Nb8-c6 Bf1-b5 Ng8-f6 O-O Nf6xe4 Rf1-e1 Ne4-d6
Nf3xe5 Bf8-e7 Bb5-f1 Nc6xe5 Re1xe5 O-O d2-d4 Be7-f6 Re5-e1 Rf8-e8 Bc1-f4
Re8xe1 Qd1xe1 Nd6-e8 c2-c3 d7-d5 Nb1-d2 Bc8-f5 a2-a4 a7-a5 Qe1-e3 Bf6-e7
Ra1-e1 Be7-d6 Nd2-f3) -0.34/30 13} 2. d4 exd4 {(e5xd4 Qd1xd4 Nb8-c6 Qd4-a4
Ng8-f6 Nb1-c3 Bf8-b4 Bc1-d2 O-O O-O-O Rf8-e8 f2-f3 Bb4-c5 Bd2-g5 h7-h6
Bg5-h4 Nc6-e5 Nc3-d5 g7-g5 Nd5xf6+ Qd8xf6 Bh4-g3 c7-c6) +0.13/20 2} 3. Qxd4
Nc6 {(Nb8-c6 Qd4-e3 Ng8-f6 Bc1-d2 Bf8-e7 Nb1-c3 d7-d5 Nc3xd5 Nf6xd5 e4xd5
Qd8xd5 Ng1-e2 Bc8-g4 Ne2-f4 Qd5-d6 f2-f3 Bg4-f5 O-O-O O-O-O g2-g4 Be7-g5
g4xf5 Rh8-e8 Qe3-d3 Bg5xf4) +0.33/21 3} 4. Qe3 Nf6 {(Ng8-f6 Bc1-d2 Bf8-e7
Nb1-c3 d7-d5 e4xd5 Nf6xd5 Nc3xd5 Qd8xd5 Ng1-e2 Bc8-g4 Ne2-c3 Qd5-h5 f2-f3
Bg4-e6 O-O-O O-O-O Bf1-d3 Be7-f6 Nc3-b5 Qh5-e5 Qe3xe5 Nc6xe5) +0.35/23 3}
5. e5 Ng4 {(Nf6-g4 Qe3-e2 d7-d6 e5xd6+ Bc8-e6 Ng1-f3 Qd8xd6 h2-h3 Ng4-f6
Nb1-c3 O-O-O Bc1-d2 Nc6-b4 O-O-O Nb4xa2+ Nc3xa2 Be6xa2 Bd2-e3 Qd6xd1+
Qe2xd1 Rd8xd1+ Kc1xd1 Ba2-d5 Bf1-d3 Bf8-e7 Rh1-e1 Rh8-d8) +1.11/23 4} 6.
Qe2 d6 {(d7-d6 f2-f3 Ng4xe5 f3-f4 Nc6-d4 Qe2-e4 Qd8-h4+ g2-g3 Qh4-e7 f4xe5
d6xe5 Bf1-h3 f7-f5 Qe4-d3 Qe7-c5 c2-c3 Qc5-c6 c3xd4 Qc6xc1+ Ke1-f2 Qc1xb2+
Ng1-e2 Qb2xa1 Qd3-b3 e5xd4) +1.12/21 4} 7. exd6+ Be6 {(Bc8-e6 Ng1-f3 Qd8xd6
h2-h3 Ng4-f6 Nb1-c3 O-O-O Bc1-d2 Nc6-b4 O-O-O Nb4xa2+ Nc3xa2 Be6xa2 Qe2-e3
Kc8-b8 Bf1-b5 Ba2-d5 Rh1-e1 Qd6-c5 Bb5-a4 Bf8-d6 Nf3-e5 Qc5xe3 Re1xe3 c7-c6
g2-g4 Rh8-e8 f2-f4) +1.14/23 4} 8. Qb5 Bxd6 {(Bf8xd6 Ng1-f3 Qd8-e7 Bf1-e2
O-O-O Nb1-c3 a7-a6 Qb5-g5 f7-f6 Qg5-h5 Nc6-b4 O-O Nb4xc2 Ra1-b1 Kc8-b8
a2-a3 Ng4-e5 Qh5-h4 Be6-f5 Qh4-a4 Ne5xf3+ Be2xf3 Qe7-e5) +2.80/23 9} 9. Qe2
Bc5 {(Bd6-c5 Bc1-e3 Nc6-d4 Qe2-d2 Ng4xe3 f2xe3 Nd4-f5 Qd2xd8+ Ra8xd8 e3-e4
Nf5-e3 Bf1-d3 Ne3xg2+ Ke1-d2 Bc5-e3+ Kd2-e2 Be6-g4+ Ng1-f3 Be3-b6 Rh1-f1
O-O Ke2-d2 Ng2-e3 Rf1-e1 Bg4xf3 Re1xe3 Bb6xe3+ Kd2xe3 Bf3-h5 Nb1-c3)
+4.82/21 3} 10. f3 Bf2+ {(Bc5-f2+ Qe2xf2 Ng4xf2 Ke1xf2 Qd8-d4+ Bc1-e3
Qd4xb2 Bf1-b5 Qb2xa1 Ng1-e2 O-O a2-a4 Rf8-d8 Rh1-c1 Nc6-e5 Nb1-c3 Qa1-a3
Be3-f4 a7-a6 Bb5xa6 Ra8xa6 Bf4xe5 Rd8-d2) +6.27/21 4 0-1 . Arena
Adjudication. Illegal move!} 11. Qxf2 0-1

Why illegal move? Seems to be a bug.

**Comment by dav1312** [comment]
I don't see any illegal move
The PGN but cleaned:
```
1. e4 e5 2. d4 exd4 3. Qxd4 Nc6 4. Qe3 Nf6 5. e5 Ng4 6. Qe2 d6 7. exd6+ Be6
8. Qb5 Bxd6 9. Qe2 Bc5 10. f3 Bf2+ {0-1 . . Arena Adjudication. Illegal
move!} 11. Qxf2 *
```
